roles/validate-input-k8s-cluster/files/validate_k8s_objects.py

```diff
@@ -112,6 +140,45 @@ def main():
             if each_sa not in apigee_secrets:
                 validations.append(f"Service Account Secret: {each_sa} Not found in {apigee_namespace} namespace")  # noqa
 
+        apigee_components = ["mart", "runtime","udca", "logger", "metrics", "watcher","cassandra"]
+        project_id = input_data.get('gcp','').get('projectID','')
+        success, roles_response = get_roles(project_id)
+        
+        if success:
+            roles_map_file_path = 'roles_map.json'
+            roles_map = {}
+            try:
+                with open(roles_map_file_path, 'r') as file:
+                    roles_map = json.load(file)
+            except FileNotFoundError:
+                validations.append(f"ERROR: Couldn't perform kubernetes validations. Reason: File '{roles_map_file_path}' not found.")
+            
+            bindings = roles_response.json()['bindings']
+            for component in apigee_components:
+                if component == "cassandra":
+                    if input_data.get(component, {}).get('backup', '').get('enabled') == True: 
```

Comment:
@payaljindal : this can be replaced by  

`if input_data.get(component, {}).get('backup', '').get('enabled') == True: `


`if input_data.get(component, {}).get('backup', '').get('enabled'):`